workflow.md

```diff
@@ -54,6 +54,12 @@ project manager would do it himself). Do not merge your own PRs!
 Please describe in PR what have you done, if it is tested or not, any additional
 information.
 
+## Merging Pull requests
+
+After review by another team member, the teammate merging the PR should merge
+`master` into the Pull request's branch *without* the `--no-ff` option, and then
```

Comment:
It doesn't matter how you merge into pull request's branch, because fast-forward can't happen there anyway.